docs/orchestration/concepts/api.md

```diff
@@ -63,28 +59,24 @@ client.graphql(
 
 ### Authenticating the Client with Cloud <Badge text="Cloud"/>
 
-If using Prefect Cloud, authentication is required. For interactive use, the most common way to use the Cloud Client is to generate a `USER`-scoped token and provide it to the client. After doing so, users can save the token so it persists across all Python sessions:
+If using Prefect Cloud, authentication is required. For interactive use, the most common way to use the Cloud Client is to generate a user-based API key and provide it to the client. After doing so, users can save the key so it persists across all Python sessions:
 
 ```python
 import prefect
-client = prefect.Client(api_token="YOUR_USER_TOKEN")
+client = prefect.Client(api_token="YOUR_USER_KEY")
```

Comment:
Maybe here (and elsewhere) we should call this `"API_KEY"` instead

Reply:
is it confusing that both user and service accounts are now using API keys? Or do you think there's enough context that you can decide which one should be used?

Reply:
Can we do `API_KEY` and `SERVICE_ACCOUNT_API_KEY` where necessary? It seems like having the default be the user api key and the service account ones be the special case would help with clarity. Service accounts are just special users 🤷

Reply:
that seems like a good heuristic to follow! 👍